package pubsub;

//订阅主题
message CSSubscribeTopicReq
{
  required bytes topic = 1;
}

message CSSubscribeTopicRsp
{
  required uint32 status_code = 1;
}

//发布主题
message CSPublishTopicReq
{
  required bytes topic = 1;
  required bytes content = 2;
}

message CSPublishTopicRsp
{
  required uint32 status_code = 1;
}

//推送给所有订阅者
message CSPublishTopicNtf
{
  required bytes topic = 1;
  required bytes content = 2;
}
